guides/v2.0/extension-dev-guide/test-module.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+group: extension-dev-guide
+subgroup: 
+title: Test your component
+menu_title: Test your component
+menu_order: 2
+menu_node: 
+github_link: extension-dev-guide/test-module.md
+
+---
+
+##{{page.menu_title}}
+
```

Comment:
Pretty minor but the TOC doesn't match the sections right now. Try adding after {{page.menu_title}}:

```
* TOC
{:toc}
```